docs/interop.md

```diff
@@ -98,7 +98,7 @@ SetString(MarshalString.GetAbi(marshalStr));
 **Note:** The CLR still supports marshaling COM (IUnknown), but not WinRT (IInspectable), interop interfaces. There are two approaches to marshaling IInspectable interfaces in C#/WinRT. 
 
 ##### Projected
-If possible, the interop interface should be defined in IDL and a C#/WinRT projection produced for it. This automatically generates all marshaling logic so that calling code can pass and receive projected types. For an example of this, see the [IUserConsentVerifierInterop definition](https://github.com/microsoft/CsWinRT/blob/master/TestComponentCSharp/TestComponentCSharp.idl#L356) and [related test code](https://github.com/microsoft/CsWinRT/blob/master/UnitTest/TestComponentCSharp_Tests.cs#L1039).
+If possible, the interop interface should be defined in IDL and a C#/WinRT projection produced for it. This automatically generates all marshaling logic so that calling code can pass and receive projected types. For an example of this, see the [IUserConsentVerifierInterop definition](https://github.com/microsoft/CsWinRT/blob/master/src/Tests/TestComponentCSharp/TestComponentCSharp.idl#L360) and [related test code](https://github.com/microsoft/CsWinRT/blob/master/src/Tests/UnitTest/TestComponentCSharp_Tests.cs#L1092).

```

Comment:
Those line numbers are probably going to change often, maybe we should make a copy of the example here rather than references.

Reply:
yeah you're right, just made an update